### DISCUSSION TITLE
Issues with publishing extension for 32 bit Windows

### DISCUSSION THREAD

**Discussion by mmueller2012** [microsoft/vscode-discussions#510] [Extension Development QnA]
When publishing the 32 bit Windows version of my extension to the store, I get an email about potential issues, followed by another copy of this mail after 2 days and finally a failure email after about 4-5 days. The linked zip file consists of only one file `1_VirusScan.log` with the text `Something went wrong, please retry after sometime.`.

This doesn't happen on every release and is limited to Windows 32 bit. All other 9 architectures, including 64 bit Windows, publish just fine. The affected extension is [SemanticDiff](https://marketplace.visualstudio.com/items?itemName=semanticdiff.semanticdiff). Does anyone have an idea how to solve this issue? It would be a shame if I had to drop Windows 32 bit support due to this issue :-/.

**Comment by isidorn** [comment]
Thanks for opening this discussion, but can you please open a new issue on the Marketplace https://github.com/microsoft/vsmarketplace
And feel free to ping me @isidorn on the issue. Thanks!
